### ISSUE TITLE
Building for "Any iOS Device (arm64)" fails

### ISSUE THREAD

**Issue by skohrt** [chicio/RangeUISlider#51]
Hello Fabrizio,
I used your really fantastic RangeUISlider in one of my small iOS Apps.
I installed it as Package Dependency in Xcode 13.3 under Monterey 12.2.1 (Version rule: master).
My app with your slider works properly on different connected iPhones as well as on different Xcode iPhone simulators. No errors at all.
But as soon as I try to "Archive" it, the build for "Any iOS Device (arm64)" fails with 49 issues.
I'm pretty sure, that it is not a bug in your RangeUISlider. Do you have any clue, what might goes wrong? I'm running out of ideas.
I attached the list of issues and an error log file.
Thank you very much for your help.
All the best, Stephan
[Compiler Error List.pdf](https://github.com/chicio/RangeUISlider/files/8310173/Compiler.Error.List.pdf)
[Build target RangeUISlider_2022-03-19T20-44-35.txt](https://github.com/chicio/RangeUISlider/files/8310174/Build.target.RangeUISlider_2022-03-19T20-44-35.txt)

**Comment by chicio** [comment]
Hi @skohrt ,

thank you for your interest in RangeUISlider. Yep you're right, there is some problem with installing RangeUISlider as package dependecy. Basically I was still pointing to iOS 9, and for the package manager this is not good: basically the swift build toolchain fails to import SwiftUI classes for a target so low.
Given the fact that now I support also SwiftUI, I increased the target to iOS 13 in the Package.swift on master. I will release a new version with this modification (I tested it and it works as expected).

Thank you for your support and remember to ⭐ star the repository ⭐ 

Fabrizo.
